manual/core/integration/README.md

```diff
@@ -562,6 +562,7 @@ Here are the recommended TinkerPop versions for each driver version:
 
 <table>
 <tr><th>Driver version</th><th>TinkerPop version</th></tr>
+<tr><td>4.14.1</td><td>3.5.3</td></tr>
```

Comment:
It seems a little strange for Tinkerpop to get a minor version bump for only a patch release for the Java driver... not sure what I think about this yet.

Reply:
Hmm indeed. But if the goal is to fix CVEs I guess we don't have much of a choice. (Besides we are doing the same for Jackson).